src/components/BenefitPlanSearcher.js

```diff
@@ -0,0 +1,217 @@
+import React, {useState, useEffect, useRef} from "react";
+import {injectIntl} from "react-intl";
+import {
+    withModulesManager,
+    formatMessage,
+    formatMessageWithValues,
+    Searcher,
+    formatDateFromISO,
+    coreConfirm,
+    journalize,
+    withHistory,
+    historyPush,
+} from "@openimis/fe-core";
+import {bindActionCreators} from "redux";
+import {connect} from "react-redux";
+import {fetchBenefitPlans, deleteBenefitPlan} from "../actions";
+import {
+    DEFAULT_PAGE_SIZE,
+    ROWS_PER_PAGE_OPTIONS,
+    EMPTY_STRING,
+    RIGHT_BENEFIT_PLAN_UPDATE,
+    RIGHT_BENEFIT_PLAN_DELETE,
+} from "../constants";
+import {IconButton, Tooltip} from "@material-ui/core";
+import EditIcon from "@material-ui/icons/Edit";
+import DeleteIcon from "@material-ui/icons/Delete";
+import BenefitPlanFilter from "./BenefitPlanFilter";
+
+const BenefitPlanSearcher = ({
+                                 intl,
```

Comment:
I wonder if the 'indentation' is quite too big in that case. Please verify this and fix in case of potential issue.

Reply:
done